src/code_1A5830.c

```diff
@@ -274,20 +285,37 @@ ApiStatus SetStatusTable(ScriptInstance* script, s32 isInitialCall) {
 ApiStatus SetIdleAnimations(ScriptInstance* script, s32 isInitialCall) {
     Bytecode* args = script->ptrReadPos;
     ActorID actorID = get_variable(script, *args++);
-    s32 var1;
+    s32 partIndex;
     s32 var2;
 
     if (actorID == ActorID_SELF) {
         actorID = script->owner1.actorID;
     }
 
-    var1 = get_variable(script, *args++);
+    partIndex = get_variable(script, *args++);
     var2 = get_variable(script, *args++);
-    get_actor_part(get_actor(actorID), var1)->idleAnimations = var2;
+    get_actor_part(get_actor(actorID), partIndex)->idleAnimations = var2;
     return ApiStatus_DONE2;
 }
 
-INCLUDE_ASM(s32, "code_1A5830", func_8027CC10);
+ApiStatus func_8027CC10(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    ActorID actorID = get_variable(script, *args++);
+    s32 partIndex;
+
+    if (actorID == ActorID_SELF) {
+        actorID = script->owner1.actorID;
+    }
+
+    partIndex = get_variable(script, *args++);
+
+    // weirdly unused
```

Comment:
if they had more vars that went unused, they still have to advance past the bytecode so that things don't get out of sync

Reply:
They could just do *args++ for that, but maybe they used get_variable since it's a macro